### PR TITLE
SC 2021 Artifact Description/Evaluation 

### PR DESCRIPTION
**Pull request by vahldiek**
This PR includes the call for AE + results + report for SuperComputing 2021. Going forward we would like to continue hosting SC21 AD/AE results on sysartifacts.

### CODE REVIEW

**Comment by ricci** [comment]
I just want to verify: the ACM badges used on the Results pages are there because SC did use the ACM badging standards, right? And not just because they were copied from another template?

**Comment by ricci** [comment]
Minor nit, but I'd suggest making the column of the final title on the results page 'Artifact' to avoid it linewrapping the table header.

File: sc2021/results.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+**Evaluation Results**:
+
+* 66 Artifact Available (+2 rejected major revision)
+* 52 Artifact Functional (+2 rejected major revision)
+* 38 Results Reproduced (+2 rejected major revision)
+
+**SC21 Best Reproducibility Advancement Award Finalists**:
+
+* Weihao CuiHan ZhaoQuan ChenNingxin ZhengJingwen LengJieru ZhaoZhuo SongTao
```
Comment (ricci) [inline sc2021/results.md:20]: Looks like spaces got lost in the author lists via cut and paste

**Comment by vahldiek** [comment]
> I just want to verify: the ACM badges used on the Results pages are there because SC did use the ACM badging standards, right? And not just because they were copied from another template?

Yes. SC alternates publishers (ACM/IEEE). We tried to align ACM and IEEE badges/systems as much as possible, but this year is based on ACM and its capabilities in the ACM DL. You can find the badges already in the proceedings (https://dl.acm.org/doi/proceedings/10.1145/3458817) including ACM DL entries for papers with artifacts (some artifacts are directly stored in ACM DL others link to DOIs on Zenodo).

Thanks for catching the space and line-break issue. I squashed the changes into the last commit.

**Pull request merged**